src/teleop/gui/src/components/AutonTask.vue

```diff
@@ -94,12 +88,25 @@ export default {
       messageType : 'sensor_msgs/NavSatFix'
     });
 
-    this.localization_sub = new ROSLIB.Topic({
+    this.tfClient = new ROSLIB.TFClient({
       ros : this.$ros,
-      name : '/imu/data',
-      messageType : 'sensor_msgs/Imu'
+      fixedFrame : 'odom',
+      angularThres : 0.01,
+      transThres : 0.01
+    });
+
+    this.tfClient.subscribe('base_link', (tf) => {
+        console.log(tf)
```

Comment:
Maybe don't console log unless this is not gonna run super often? (But I assume it would)